src/components/App/App.js

```diff
@@ -75,25 +68,55 @@ const App = () => {
     }
     );
   }, []);
+  const resetResultState = () => {
+    setResultState([])
+  }
+  const displayInfo = (e) => {
+    console.log(e)
+    clickImage(true)
+    const singleChampion = appState.filter((champion) => {
+      return champion.id.includes(e.target.id)
+    })
+    setSingleInfo(singleChampion)
+    console.log(singleChampion)
+  }
 
-  const displayInfo = (event) => {
-    event.preventDefault();
-    const getChampion = appState.champions.filter((champion) => {
-      return champion.title === event.target.id
+  const findChampions = (e) => {
+    e.preventDefault();
+    console.log(e)
+    const getChampion = appState.filter((champion) => {
+      return champion.tags.includes(e.target.name)
     })
-    setSingleInfo({ champions: getChampion })
-    console.log(singleInfo)
-  };
+    setResultState(getChampion)
+  }
+
 
   return (
     < div className="App" >
-      <h1 className="welcome-message">Lane In Your League</h1>
-      < div className="champion-load">
-        <h2>
-          <ChampionsLoad characters={appState} displayInfo={displayInfo} />
-        </h2>
+      <div className="filter-button">
+        <TagForm findChampions={findChampions} resetResultState={resetResultState} />
       </div>
-    </div >
+      <h1 className="welcome-message">Lane In Your League</h1>
+      {singleInfo.length ? <div className="sinlge-champion"><ChampionInfo champion={singleInfo[0]} /></div> :
+        <div className="champion-container">
+          {!resultState.length ?
```

Comment:
I was asking Leta what the convention was for formatting ternaries like this, and they said that, if a ternary has to be broken up into multiple lines due to length, it's better to refactor the expression as an if/else statment (just for claritie's sake)